docs/docs/getting-started/headless.md

```diff
@@ -0,0 +1,98 @@
+---
+id: headless
+title: Headless
+sidebar_label: Headless
+slug: /getting-started/headless
+---
+
+You can also run React Native Skia on Node.
```

Comment:
Should we make the intro contain a bit more info? Like first an explanation to what the headless concept is, and then how Node and RN Skia fits into the headless concept - and then the example?